extensions/standard-processors/tests/CPPLINT.cfg

```diff
@@ -0,0 +1 @@
+filter=-build/namespaces
```

Comment:
As per my previous comment maybe we could go with `build/namespaces_literals` here as well.

Reply:
with the filter `build/namespaces_literals ` in the root CPPLINT.cfg this file became obsolete deleted it in https://github.com/apache/nifi-minifi-cpp/pull/1225/commits/78f8658fde802c73a5965dd734a544acbc37dca3